data_prep/scotland/get_reference_layers.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+set -x
+OUT=out_layers
+mkdir -p $OUT
+
+function route_network {
+  # From https://github.com/nptscot/outputdata/releases
+  echo "Manually download https://github.com/nptscot/outputdata/releases/download/v2025-02-01/rnet_2025-02-01.pmtiles and save it to $OUT/route_network.pmtiles"
```

Comment:
Private GH repo, sorry :(  But the data is public

Reply:
Oh wait, I think I was confused. Based on your README updates, it looks like you've already done this by copying it onto od2net.org.

I've rebased #108 onto this branch to account for this and will push it up as soon as I've tested it.